### ISSUE TITLE
Getting started Blog index doesn't create an index

### ISSUE THREAD

**Issue by goetzk** [torchbox/wagtail#3096]
Hi,
There seems to be some text missing from the tutorial

* Visit http://docs.wagtail.io/en/v1.6.3/getting_started/tutorial.html#blog-index
* Follow steps
* Pages in your site are not listed on the index.

I suggest there should be a short snippet for the template which iterates and lists all pages in the site to add content for the index page.

thanks,

**Comment by gasman** [comment]
Hi @goetzk,

The following section of the tutorial ("Related items") covers adding items to the index page, as manually-assigned links. It would probably be a good idea for the tutorial to cover creating automatic listings too, but that's a separate task.

**Comment by shacker** [comment]
I know that this is already closed, but just wanted to weigh in as I just experienced the same. Having blog pages automatically included on a blog index page is almost fundamental to the definition of a blog and it is certainly what a person expects when going through a "basic blog" tutorial. My suggestion would be to refactor the getting started tutorial to build something more like a standard blog (perhaps with categories/tags and date-based URLs, etc.)

I would suggest removing the RelatedLinks section of Getting Started, as the modeling is actually pretty abstract and hard to follow for someone just learning wagtail for the first time. And it doesn't explain the logic of the interrelated models LinkFields, RelatedLink, BlogIndexRelatedLink, and how those are combined and factored into BlogIndexPage. It's all pretty confusing. It's like the intro tutorial goes from 0-60 in 3 seconds, leaving the user confused as to why they still don't have basic blog functionality. 

Not wanting to be too critical, but I really do think there's a lot of room for improvement here. 

I might be willing to contribute in this area, time permitting.

**Comment by goetzk** [comment]
Hi @gasman ,
@shacker 's first paragraph really sums up what I was expecting that bit of the tutorial to be, hence my assumption that there was something missing. 
@shacker has said everything else I could have added so will leave it at that.

Thanks both!
kk

**Comment by zerolab** [comment]
Hi @goetzk @shacker,

Your feedback is appreciated. IMHO the best form of feedback would be a pull request with the relevant changes as it can be easy to lose sight of the basic tutorial if you are not doing it every day.

Thank you.

**Comment by shacker** [comment]
Great thanks - I'll have a PR ready today or later this week.
